docs/Developers/Git_Basics.rst

```diff
@@ -245,6 +245,12 @@ the desired branch:
     $ git fetch origin
     $ git rebase origin/master
 
+.. note::
+
+    ``origin`` is the remote of https://github.com/coala/coala
+    or https://github.com/coala/coala-bears. It is not the remote
+    of your fork.
```

Comment:
This could be phrased better I think:
> Note: This assumes that the remote ``origin`` is the original coala repository at https://github.com/coala/coala (or other, like coala/coala-bears, etc) and **not** your fork. If you're unsure about this, run `git remote -v` to check which remote points to the original repository and use it instead of ``origin`` in the following section.

Also I'd put this at the very top of the Rebasing section